### PR TITLE
AutoFix Logo Size - KPIs should not be required

### PR DESCRIPTION
**Pull request by rsamanez**
AutoFix Logo size.
 - Setting logo aspect ratio 3:1
 
KPIs should not be required
 - you can left the kpi array empty in the JSON configuration file. It will hide the KPI banner

### CODE REVIEW

**Comment by rsamanez** [comment]
Hi Jim, please check this pull request.

File: nerdlets/pathpoint-nerdlet/components/Header/Header.js [hunk 1]
```diff
@@ -56,55 +56,57 @@ const Header = ({
...
-                  kpi.link !== '' && window.open(kpi.link);
-                }}
-                className="kpicontent"
-              >
-                <div className="kpicontent--colorgrey kpicontent--size10">
-                  {kpi.shortName}
-                </div>
-                <div className="kpicontent--colorblack kpicontent--size12">
-                  {PrintKPI(kpi)}
+      {filterKpis.length > 0 && (
```
Comment (JimHagan) [inline nerdlets/pathpoint-nerdlet/components/Header/Header.js:59]: @rsamanez What is a valid JSON look like now.

Is this valid?

``` 
{
    "pathpointVersion": "1.5.21",
    "kpis": [],
     "stages": [
         //stage data      
      ]
```


What about a scenario where the kpis is missing completely?

```
{
    "pathpointVersion": "1.5.21",
    // NO KPIS SECTION
     "stages": [
         //stage data      
      ]
```

**Comment by JimHagan** [comment]
I left some questions in a review.  Also what is the empty file `nerdlets/pathpoint-nerdlet/.env.json`??? Should it be deleted?

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "scripts": {
         "start": "nr1 nerdpack:serve",
         "samanez": "nr1 nerdpack:serve --profile=samanez",
-        "test": "jest -w 1",
+        "test": "jest -w 1 -u",
```
Comment (JimHagan) [inline package.json:8]: @rsamanez This change is not mentioned in the pull request notes.

File: nerdlets/pathpoint-nerdlet/components/Header/styles.scss [hunk 3]
```diff
@@ -18,20 +18,20 @@
...
-  justify-content: center;
+  justify-content: flex-start;
   align-items: center;
   margin-left: 10px;
 }
 
 .logo-container img {
-  width: 82px !important;
-  height: auto;
+  width: auto !important;
```
Comment (JimHagan) [inline nerdlets/pathpoint-nerdlet/components/Header/styles.scss:29]: Do you have a screenshot of how this appears in the UI now?

**Review by JimHagan — CHANGES_REQUESTED** [review]

File: nerdlets/pathpoint-nerdlet/components/Header/Header.js (hunk 1 above)
Comment (rsamanez) [reply nerdlets/pathpoint-nerdlet/components/Header/Header.js:59]: You have to keep an empty array to be valid.
{
    "pathpointVersion": "1.5.21",
    "kpis": [],
     "stages": [
         //stage data      
      ]

The second smple IS NOT A VALID SCHEMA.

File: package.json (hunk 2 above)
Comment (rsamanez) [reply package.json:8]: This is an Update to resolve an issue with the Unit tests

Comment (rsamanez) [reply package.json:8]: Using this Sample Logo from this URL:
https://www.freepnglogos.com/uploads/google-logo-png/google-logo-png-suite-everything-you-need-know-about-google-newest-0.png

Results:
image

Using this other Logo:
https://www.freepnglogos.com/uploads/google-logo-png/file-google-logo-svg-wikimedia-commons-23.png

Results:
image

File: nerdlets/pathpoint-nerdlet/components/Header/styles.scss (hunk 3 above)
Comment (rsamanez) [reply nerdlets/pathpoint-nerdlet/components/Header/styles.scss:29]: Using this Sample Logo from this URL:
https://www.freepnglogos.com/uploads/google-logo-png/google-logo-png-suite-everything-you-need-know-about-google-newest-0.png

Results:
image

Using this other Logo:
https://www.freepnglogos.com/uploads/google-logo-png/file-google-logo-svg-wikimedia-commons-23.png

Results:
image

**Comment by rsamanez** [comment]
.env.json file is not empty

image

File: nerdlets/pathpoint-nerdlet/components/Header/Header.js (hunk 1 above)
Comment (JimHagan) [reply nerdlets/pathpoint-nerdlet/components/Header/Header.js:59]: Okay.  We'll want to make a note in the KPIs documentation.  I will add a small change to the branch which updates the KPI docs.

**Comment by JimHagan** [comment]
> 

As soon as that file is deleted I will merge this branch.

**Review by JimHagan — APPROVED** [review]

**Pull request merged**